site/assets/scss/_alerts.scss

```diff
@@ -1,4 +1,4 @@
-.bd-ods-incompatibility-alert {
+.bd-ods-alert {
   .alert-heading ~ p {
```

Comment:
I can't see any use case of this selector, should we remove this ? Or at least change this selector if needed.